package.json

```diff
@@ -83,6 +84,7 @@
     "eslint": "8.10.0",
     "eslint-plugin-import": "2.25.4",
     "eslint-plugin-local": "1.0.0",
+    "get-stream": "6.0.1",
```

Comment:
FYI, this was already in node_modules, a dependency of web-ext and ts-jest, and a few others.